HomeHospital-Backend/src/models/medicalFacility.Model.js

```diff
@@ -21,14 +22,15 @@ const medicalFacility = new mongoose.Schema({
 	phoneNumber: {
 		type: String,
 		default: null,
+		validate:[validator.isMobilePhone, 'Please enter a Valid Phone Number']
```

Comment:
Need to correct the validation for the phone number so it works in the same way as the other one @bushx